tools/accuracy_checker/accuracy_checker/metrics/detection.py

```diff
@@ -683,22 +702,71 @@ def evaluate(self, annotations, predictions):
         accuracy = true_positive / (true_positive + false_positive)
         return [accuracy]
 
-    @staticmethod
-    def calc_iou(gt_box, dt_box):
-        # Convert detected face rectangle to integer point form
-        gt_box = list(map(lambda x: int(round(x, 0)), gt_box))
-        dt_box = list(map(lambda x: int(round(x, 0)), dt_box))
 
-        # Calculate overlapping width and height of two boxes
-        inter_width = min(gt_box[2], dt_box[2]) - max(gt_box[0], dt_box[0])
-        inter_height = min(gt_box[3], dt_box[3]) - max(gt_box[1], dt_box[1])
+class AVAHeadAccuracy(FullDatasetEvaluationMetric):
 
-        if inter_width <= 0 or inter_height <= 0:
-            return None
+    __provider__ = 'ava_head_accuracy'
```

Comment:
what the difference between this metric calculation approach and youtube faces accuracy metric?

Reply:
I've applied your feedback and reused youtube faces metric instead.